src/plugins/navigation/public/top_nav_menu/_index.scss

```diff
@@ -12,3 +12,12 @@
 .kbnTopNavMenu__badgeGroup {
   margin-right: $euiSizeM;
 }
+
+.kbnTopNavMenu__betaBadgeWrapper {
+  display: flex;
+  align-items: center;
+}
+
+.kbnTopNavMenu__betaBadgeItem {
+  margin-right: $euiSizeS;
+}
```

Comment:
Consider updating these styles to account for the removal of the badge's wrapping `div` (per previous suggestion) and apply styles to vertically center and style the badge when hovered/focused while in the button.

```suggestion
.kbnTopNavMenu__betaBadgeItem {
  margin-right: $euiSizeS;
  vertical-align: middle;
  
  button:hover &,
  button:focus & {
    text-decoration: underline;
  }

  button:hover & {
    cursor: pointer;
  }
}
```

Reply:
You are awesome Michael!